rhobs/alerting/data_plane/prometheus.pipeline_alerts.yaml

```diff
@@ -76,7 +76,8 @@ spec:
               Tekton pipeline controller on cluster {{ $labels.source_cluster }} has appeared deadlocked on {{ $value }} pipelineruns.
             alert_team_handle: <!subteam^S04PYECHCCU>
             team: pipelines
-            runbook_url: TBD
+            # link will be available once https://gitlab.cee.redhat.com/konflux/docs/sop/-/merge_requests/163 merges
+            runbook_url: https://gitlab.cee.redhat.com/konflux/docs/sop/-/blob/main/pipeline-service/slos/tekton-pipeline-related-deadlocks.md
```

Comment:
Yes that was my intent.  With all the panel/alert/SOP activity going on across many deployments, the basic rationale I had previously discussed with Alex is that before promoting any alerts from stage to prod, their associated SOPs have been vetted/approved by the app sre tiger team and merged.

See https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/112711#note_11783344

Are you OK with that as well @yftacherzog ?

Reply:
I'm ok with that. I'd prefer the comment NOT to be there in that case (just have a dead link with no comments until that MR is merged), but it might just be me.

Reply:
OK cool (alert promote to prod etc.).

The comments in part are to assist @divyansh42, who will be taking point on this for openshift pipelines, and the various leads who will do his day to day coaching after my transfer from the team kicks in at the end of August, just in case the SOP review lingers (some of the tiger team is on leave currently).

Ideally the comments won't be there long.